RecoBTag/TensorFlow/plugins/DeepFlavourTagInfoProducer.cc

```diff
@@ -209,20 +218,16 @@ void DeepFlavourTagInfoProducer::produce(edm::Event& iEvent, const edm::EventSet
               { return btagbtvdeep::sv_vertex_comparator(sva, svb, pv); });
     // fill features from secondary vertices
     for (const auto & sv : svs_sorted) {
-      if (reco::deltaR(sv, jet) > jet_radius_) continue;
+      if (reco::deltaR2(sv.position() - pv.position(), flip_ ? -jet_dir : jet_dir) > (jet_radius_*jet_radius_)) continue;
```

Comment:
@emilbols 
just to keep a record for future references: this change is undone in #23781 
Please confirm that this change here was made as a mistake and not as a result of an update to the definitions used for new training.

Reply:
@slava77 it was just a mistake. In the training it was always done in the previous way.